test/lit_tests/generate_td.mlir

```diff
@@ -22,6 +22,7 @@
 
 // TD: module attributes {transform.with_named_sequence} {
 // TD:  transform.named_sequence @__transform_main(%[[ROOT:.+]]: !transform.any_op) {
+// CLEAN-NOT: attributes {transform.with_named_sequence}
```

Comment:
I thought there was a different pass which removed things cc @ftynse

Reply:
There is a `-test-transform-erase-schedule` (or something like that), but it won't remove the attribute.